projects/swimlane/ngx-ui/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## HEAD (unreleased)
 
+## 45.0.2 (2023-7-20)
+
+- Fix (`json-editor-helper`): Repair dataTypeMap overwrite
```

Comment:
Is there any way to make this a bit more generic or understandable as to what was fixed?

Reply:
what do you think about 'fix input type overwrite'?

Reply:
Yeah that sounds better. Let's go with that.